func/server/server.go

```diff
@@ -37,34 +37,49 @@ const (
 	wrapperServerImageEnv = "WRAPPER_SERVER_IMAGE"
 )
 
-var (
-	port                       = flag.Int("port", 9445, "The server port")
-	functions                  = flag.String("functions", "./functions", "Path to cached functions.")
-	config                     = flag.String("config", "./config.yaml", "Path to the config file.")
-	enablePrivateRegistries    = flag.Bool("enable-private-registries", false, "if true enables the use of private registries and their authentication")
-	registryAuthSecretPath     = flag.String("registry-auth-secret-path", "/var/tmp/config-secret/.dockerconfigjson", "The path of the secret used for authenticating to custom registries")
-	registryAuthSecretName     = flag.String("registry-auth-secret-name", "auth-secret", "The name of the secret used for authenticating to custom registries")
-	enablePrivateRegistriesTls = flag.Bool("enable-private-registries-tls", false, "if enabled, will prioritize use of user provided TLS secret when accessing registries")
-	tlsSecretPath              = flag.String("tls-secret-path", "/var/tmp/tls-secret/", "The path of the secret used in tls configuration")
-	podCacheConfig             = flag.String("pod-cache-config", "/pod-cache-config/pod-cache-config.yaml", "Path to the pod cache config file. The file is map of function name to TTL.")
-	podNamespace               = flag.String("pod-namespace", "porch-fn-system", "Namespace to run KRM functions pods.")
-	podTTL                     = flag.Duration("pod-ttl", 30*time.Minute, "TTL for pods before GC.")
-	scanInterval               = flag.Duration("scan-interval", time.Minute, "The interval of GC between scans.")
-	disableRuntimes            = flag.String("disable-runtimes", "", fmt.Sprintf("The runtime(s) to disable. Multiple runtimes should separated by `,`. Available runtimes: `%v`, `%v`.", execRuntime, podRuntime))
-	functionPodTemplateName    = flag.String("function-pod-template", "", "Configmap that contains a pod specification")
-)
+type options struct {
+	// The server port
+	port int
+	// The runtime(s) to disable. Multiple runtimes should separated by `,`.
+	disableRuntimes string
+
+	// Parameters of ExecEvaluator
+	exec internal.ExecutableEvaluatorOptions
+	// Parameters of PodEvaluator
+	pod internal.PodEvaluatorOptions
```

Comment:
Pedantic, so your call if you fix it, but o.pod.wrapperServerImage below were a bit  confusing whtether this is actual pod info, or something else. Maybe podeval?

Reply:
I am fine with renaming it. Would "WrapperServerConatinerImageName" sound better? Should we also rename the WRAPPER_SERVER_IMAGE envvar where the actual name coming from? The later is a breaking change so probably not. Btw., there is a comment next to the field in the PodEvaluatorOptions struct (that IDEs will find and display) that tries to explain the semantics of the field.